code/ndarray.c

```diff
@@ -262,7 +262,7 @@ mp_bound_slice_t generate_slice(mp_uint_t n, mp_obj_t index) {
     // micropython seems to have difficulties with negative steps
```

Comment:
`mp_obj_slice_indices` makes the comment above redundant, I believe. If that is true, we should remove it.